contrib/google.golang.org/grpc/stats_client.go

```diff
@@ -11,6 +11,7 @@ import (
 
 	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
 	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+
```

Comment:
Since we're not changing any code in this file, it doesn't make too much sense to have this extra line as part of this PR. Feel free to do it in a separate PR though!

Reply:
No idea how this happened